tools/ci_connector_ops/ci_connector_ops/pipelines/pipelines/connectors.py

```diff
@@ -46,6 +46,9 @@ async def run_report_complete_pipeline(dagger_client: dagger.Client, contexts: L
     This is to denote when the pipeline is complete, useful for long running pipelines like nightlies.
     """
 
+    if not contexts:
```

Comment:
It's fixing a bug that lead to failed pipeline execution when no connector code was changed on a PR.